build.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const fs = require('fs');
+const https = require('https');
+const archiver = require('archiver');
+
+https.get('https://uplift.shipit.staging.mozilla-releng.net/coverage/supported_extensions', res => {
+
+  let data = '';
+    
+  res.on('data', chunk => data += chunk);
+  res.on('end', () => {
+    let content = `const SUPPORTED_EXTENSIONS = ${data};`;
+
+    fs.writeFile("supported_extensions.js", content, e => {
+      if (e) {
+        console.log(e.message);
+      }
+    });
+
+    let excludeFiles = [
+      ".gitignore", ".travis.yml", "LICENSE", "README.md",
+      "package.json", "package-lock.json", "build.js"
+    ];
+
+    fs.readdir(".", (e, files) => {
+      if(e){
+        console.log(e.message);
```

Comment:
Either `throw e` or `console.error(e.message)`. If you use the second, you should also return early.